stackstorm-tuleap/actions/create_artifact.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from Tuleap.RestClient.Artifacts import Artifacts
+from Tuleap.RestClient.Connection import CertificateVerification
+from Tuleap.RestClient.Connection import Connection
+from st2actions.runners.pythonrunner import Action
+
+
+class CreateArtifact(Action):
+    def run(self, tracker_id, values_by_field):
+        requests.packages.urllib3.disable_warnings()
+
+        connection = Connection()
+        response = None
+        success = connection.login('https://'+self.config['tuleap_domain_name']+'/api/v1',
```

Comment:
Maybe it would also be more flexible if you allowed user to pass in the whole URL and not just a domain name?

Just a thought.